proof/invariant-abstract/Invariants_AI.thy

```diff
@@ -1064,8 +1064,7 @@ lemmas valid_replies_defs = valid_replies_2_def replies_with_sc_def replies_bloc
 
 abbreviation
   "fault_tcb_states st \<equiv>
-     (\<exists>ep data. st = BlockedOnSend ep data) \<or> (\<exists>ep. st = BlockedOnReply ep) \<or>
-     st = Inactive"
+     is_blocked_on_send st \<or> is_blocked_on_reply st \<or> st = Inactive"
```

Comment:
smallest-nitpick-ever: I think that `inactive st` is an abbreviation for `st = Inactive` which would make this abbreviation ever so slightly neater. But don't bother changing this unless you want to.

Reply:
I decided not to do this, mainly because `inactive` is currently very rarely used. I'll leave changing that to a future round of consolidating these discriminators.